sidechain/sidechain-storage/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "sidechain-storage"
```

Comment:
this should be renamed to its-sidechain-storage (or sidechain-db? @clangenb  what's your take on this one?)

its stands for integritee-sidechain (as far as I know at least, I may stand corrected here 😋

Reply:
`its-sidechain-storage` has redundant naming due to `its` standing for integritee-sidechain already.

I think we can simply call it `its-db`